src/load_inputs/load_inputs.jl

```diff
@@ -72,11 +72,11 @@ function load_inputs(setup::Dict,path::AbstractString)
 	return inputs
 end
 
-function is_period_map_necessary(setup::Dict, path::AbstractString, inputs::Dict)
-	ow = setup["OperationWrapping"]==1
+function is_period_map_necessary(inputs::Dict)
+	multiple_rep_periods = inputs["REP_PERIOD"] > 1
```

Comment:
Generally, OperationWrapping == 1 was used to mean that there are multiple representative periods. In several places I change this to the more obvious check.

Reply:
The function looks good, but should we avoid implicit returns? I generally find it only leads to confusion and doesn't play as nicely with some VSCode plug-ins.

Reply:
I'm neutral on this.
Pro implicit returns: "don't have code which does nothing / is not necessary" "minimalism"
Con: Potentially, worse readability. Doesn't work with VS code plugins.

I'd support an overall code-style-consistency rewrite, with something like `black`.